src/enableDisableRules.ts

```diff
@@ -18,20 +18,20 @@
 import * as ts from "typescript";
 
 import {AbstractRule} from "./language/rule/abstractRule";
-import {IOptions} from "./language/rule/rule";
 import {forEachComment, TokenPosition} from "./language/utils";
-import {RuleWalker} from "./language/walker/ruleWalker";
 import {IEnableDisablePosition} from "./ruleLoader";
 
-export class EnableDisableRulesWalker extends RuleWalker {
-    public enableDisableRuleMap: {[rulename: string]: IEnableDisablePosition[]} = {};
-
-    constructor(sourceFile: ts.SourceFile, options: IOptions, rules: {[name: string]: any}) {
-        super(sourceFile, options);
+export class EnableDisableRulesWalker {
+    private enableDisableRuleMap: {[rulename: string]: IEnableDisablePosition[]};
```

Comment:
I wanted to change that, but there are many exported functions that expect this to be an object. So this would require a larger rewrite.